.env

```diff
@@ -38,3 +38,6 @@ HDFS=2.9.2
 SPARK=master
 DOTNET=2.1
 R=3.6
+R_DOCKER_ORG=rhub
+R_DOCKER_IMAGE=ubuntu-gcc-release
+R_DOCKER_TAG=latest
```

Comment:
I'd use a single env variable called `RHUB_IMAGE`, see my comments in the docker-compose file.

Reply:
Responded below.

Reply:
The double slash in the image name is a fair point, so I see now why you need three env vars.
Nit, but I'd still use a shorter names, without the `R_ORG`, `R_IMAGE`, `R_TAG` if they don't collide with other env variables.